WcaOnRails/config/routes.rb

```diff
@@ -92,10 +92,10 @@
 
   get 'robots' => 'static_pages#robots'
 
-  get 'server-status' => 'server_status#index'
+  get 'server-status' => 'server_status#index', as: :server_status
```

Comment:
Is this `as:` and the one below actually necessary? I see them showing up in the output of `rake routes` without any work on my part:

```
                                    server_status GET    /server-status(.:format)                                                    server_status#index
                              translations_status GET    /translations/status(.:format)                                              translations#index

```